### PR TITLE
Fix offline  gulp&django compression

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: FED.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Enforce coding standard rules
+
+Gulp: Asset Bundling & Processing, Concatenating and Minifying
+Requirements Development:
+- gulpfile.js
+- package.json
+
+Commands:
+- npm install
+- npm run build
```
Comment (justinvdm) [inline FED.md:62]: Should we update `package.json` to support this, so that we can use this command once the PR lands?

File: FED.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+FED Ways of Working
+GOAL: Maintenance, Performance, and Readability.
+
+HTML
+-  We use SMACSS, BEM methodologies to make HTML modular
+
+SMACSS Structure
+https://smacss.com/book/
+
+PATH: iogt/clients/css/
```
Comment (justinvdm) [inline FED.md]: Should this say something other than `iogt`?

Nitpick: should we use `client` instead of `clients`? Not too big a deal though.

**Comment by justinvdm** [comment]
Left minor comments, looks good :)

File: FED.md (hunk 2 above)
Comment (Mitso) [reply FED.md]: Good spot, will update now.

**Pull request merged**